src/Data/List/Lazy.purs

```diff
@@ -160,6 +162,21 @@ replicateM n m
       as <- replicateM (n - one) m
       pure (cons a as)
 
+-- | Attempt a computation multiple times, requiring at least one success.
+-- |
+-- | The `Lazy` constraint is used to generate the result lazily, to ensure
+-- | termination.
+some :: forall f a. Alternative f => Z.Lazy (f (List  a)) => f a -> f (List  a)
+some v = cons <$> v <*> Z.defer (\_ -> many v)
+
+-- | Attempt a computation multiple times, returning as many successful results
+-- | as possible (possibly zero).
+-- |
+-- | The `Lazy` constraint is used to generate the result lazily, to ensure
```

Comment:
Copied it from [List.purs](https://github.com/purescript/purescript-lists/blob/master/src/Data/List.purs#L158).  But even tho `Lazy` constraint isn't used in `many`, it's used in `some`, which is used by `many`, so it's technically true.  If want better wording here please let me know and i'll update it in both places